### PR TITLE
Upgraded to binutils 2.23.2.

### PR DESCRIPTION
**Pull request by carstene1ns**
This is the beginning of a series of pull requests to upstream some of the changes in @top-sekret's fork

(split from #102)

### CODE REVIEW

**Comment by dbeef** [comment]
I don't feel competent enough to review contents of the patchfile.

Reading diff in this form is confusing, what I am looking forward to is forking binutils on pspdev and applying changes on a separate branch, like it's done with newlib.

Script changes are OK.

File: scripts/001-binutils.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+patch -p1 < ../../patches/binutils-"$BINUTILS_VERSION"-PSP.patch
+
+
+# Create and enter the build directory.
+mkdir build-psp
+cd build-psp
+
+# Configure the build.
+CFLAGS="$CFLAGS -I/opt/local/include" \
+  CPPFLAGS="$CPPFLAGS -I/opt/local/include" \
```
Comment (dbeef) [inline scripts/001-binutils.sh:23]: Missing `-Wno-error`, not sure if intended.

Comment (carstene1ns) [reply scripts/001-binutils.sh:23]: Was intended, it is passed as option now (see below).

**Comment by carstene1ns** [comment]
Yeah, I also want to get rid of these manual patch files, but currently this suffices for having an updated toolchain.

**Pull request merged**